gpustack/policies/candidate_selectors/vllm_resource_fit_selector.py

```diff
@@ -761,7 +800,7 @@ async def find_multi_worker_multi_gpu_candidates(
             return []
 
         if self._selected_gpu_workers:
-            return await self.manual_select_multi_worker_multi_gpu_candidates(workers)
+            return await self.manually_select_multi_worker_multi_gpu_candidates(workers)
```

Comment:
From an English perspective, it's fine to use "manaully" to modify select. However, I think these methods are intended to highlight that the card selection method is for **manual** purpose. 

After doing some code filtering, we can find that there are many methods starting with `manual_`. I hope that if we make any changes, it should be a single commit with `style:` prefix message.